data/abilities.csv

```diff
@@ -0,0 +1,294 @@
+id,identifier,generation_id,is_main_series
+1,stench,3,1
+2,drizzle,3,1
+3,speed-boost,3,1
+4,battle-armor,3,1
+5,sturdy,3,1
+6,damp,3,1
+7,limber,3,1
+8,sand-veil,3,1
+9,static,3,1
+10,volt-absorb,3,1
+11,water-absorb,3,1
+12,oblivious,3,1
+13,cloud-nine,3,1
+14,compound-eyes,3,1
+15,insomnia,3,1
+16,color-change,3,1
+17,immunity,3,1
+18,flash-fire,3,1
+19,shield-dust,3,1
+20,own-tempo,3,1
+21,suction-cups,3,1
+22,intimidate,3,1
+23,shadow-tag,3,1
+24,rough-skin,3,1
+25,wonder-guard,3,1
+26,levitate,3,1
+27,effect-spore,3,1
+28,synchronize,3,1
+29,clear-body,3,1
+30,natural-cure,3,1
+31,lightning-rod,3,1
+32,serene-grace,3,1
+33,swift-swim,3,1
+34,chlorophyll,3,1
+35,illuminate,3,1
+36,trace,3,1
+37,huge-power,3,1
+38,poison-point,3,1
+39,inner-focus,3,1
+40,magma-armor,3,1
+41,water-veil,3,1
+42,magnet-pull,3,1
+43,soundproof,3,1
+44,rain-dish,3,1
+45,sand-stream,3,1
+46,pressure,3,1
+47,thick-fat,3,1
+48,early-bird,3,1
+49,flame-body,3,1
+50,run-away,3,1
+51,keen-eye,3,1
+52,hyper-cutter,3,1
+53,pickup,3,1
+54,truant,3,1
+55,hustle,3,1
+56,cute-charm,3,1
+57,plus,3,1
+58,minus,3,1
+59,forecast,3,1
+60,sticky-hold,3,1
+61,shed-skin,3,1
+62,guts,3,1
+63,marvel-scale,3,1
+64,liquid-ooze,3,1
+65,overgrow,3,1
+66,blaze,3,1
+67,torrent,3,1
+68,swarm,3,1
+69,rock-head,3,1
+70,drought,3,1
+71,arena-trap,3,1
+72,vital-spirit,3,1
+73,white-smoke,3,1
+74,pure-power,3,1
+75,shell-armor,3,1
+76,air-lock,3,1
+77,tangled-feet,4,1
+78,motor-drive,4,1
+79,rivalry,4,1
+80,steadfast,4,1
+81,snow-cloak,4,1
+82,gluttony,4,1
+83,anger-point,4,1
+84,unburden,4,1
+85,heatproof,4,1
+86,simple,4,1
+87,dry-skin,4,1
+88,download,4,1
+89,iron-fist,4,1
+90,poison-heal,4,1
+91,adaptability,4,1
+92,skill-link,4,1
+93,hydration,4,1
+94,solar-power,4,1
+95,quick-feet,4,1
+96,normalize,4,1
+97,sniper,4,1
+98,magic-guard,4,1
+99,no-guard,4,1
+100,stall,4,1
+101,technician,4,1
+102,leaf-guard,4,1
+103,klutz,4,1
+104,mold-breaker,4,1
+105,super-luck,4,1
+106,aftermath,4,1
+107,anticipation,4,1
+108,forewarn,4,1
+109,unaware,4,1
+110,tinted-lens,4,1
+111,filter,4,1
+112,slow-start,4,1
+113,scrappy,4,1
+114,storm-drain,4,1
+115,ice-body,4,1
+116,solid-rock,4,1
+117,snow-warning,4,1
+118,honey-gather,4,1
+119,frisk,4,1
+120,reckless,4,1
+121,multitype,4,1
+122,flower-gift,4,1
+123,bad-dreams,4,1
+124,pickpocket,5,1
```

Comment:
I'm noticing some Abilities from Gen V over here, do we want to include these?

Reply:
I'm excluding them in the code generation.